packages/runtime/container-runtime/src/summary/summarizerTypes.ts

```diff
@@ -251,6 +251,7 @@ export type SummarizeResultPart<TSuccess, TFailure = undefined> =
 	| {
 			success: true;
 			data: TSuccess;
+			error?: undefined;
```

Comment:
Why add error if its success? I would rather check for success / failure and then check for error only on failure.

Reply:
Yeah it was just to make grabbing the error trivial. If you prefer it the other way that's fine with me